README.md

```diff
@@ -235,6 +235,86 @@ If you don't see a command prompt, try pressing enter.
 / # wget -O - <NGINX_POD_IP>
 ```
 
+# Testing
+
+## E2E testing
+
+E2E testing purpose is to validate submariner behaviour from an integration point of
+view. It needs to be executed in connection to an existing set of clusters.
+
+The E2E tests require kubeconfigs for at least 2 dataplane clusters. By dataplane
+clusters we mean clusters which are interconnected by a deployment of submariner.
+
+E2E tests identify each cluster by their associated kubeconfig context names,
+so you need to specify the -dp-context flag for each context name you want
+the E2E tests to use.
+
+
+Assuming that we have cluster1, cluster2 and cluster3 contexts, and that
+cluster1 is our broker cluster, **to execute the E2E tests** we would do:
+
+  ```bash
+  cd test/e2e
+  go test -args -kubeconfig=creds/cluster1:creds/cluster2:creds/cluster3 \
+                -dp-context cluster2 \
+                -dp-context cluster3 \
+                -ginkgo.randomizeAllSpecs
+  ```
+
+The -kubeconfig flag can be ommited .if the KUBECONFIG environment variable
```

Comment:
Sorry just one more nit - remove the period before 'if' 😄

Reply:
Oppps, sorry